### DISCUSSION TITLE
Error? Detected N missing reading(s), data may not be accurate

### DISCUSSION THREAD

**Discussion by jcam** [emporia-vue-local/esphome#99] [Q&A]
I added a template like this to send data to another system I have, but now I'm seeing errors about missing readings.
Is the esp32 not able to keep up with readings when it runs this http request? Do I need to somehow run this in a different thread? It seems able to keep up with readings when notifying home assistant, does that use a separate thread?

Thanks!

```
  - platform: total_daily_energy
    name:    "Lower BR Mini Energy WS"
    id: lower_br_mini_ws
    accuracy_decimals: 0
    power_id:  cir2
    filters: [ multiply: 3600 ]
    unit_of_measurement: "Ws"
    on_value:
      http_request.get:
        url: !lambda >-
          return (
            (std::string) "http://192.168.1.8/index.php/ecmEngine/greeneye?SN=01000001" +
            "&SC=" + to_string(id(my_time).now().hour * 3600 + id(my_time).now().minute * 60 + id(my_time).now().second) +
            "&V=" + to_string(int((id(phase_a_voltage).state + id(phase_b_voltage).state) * 5)) +
            "&c1=" + to_string(int(id(lower_br_mini_ws).state)) + ",0,0"
          ).c_str();
```

**Comment by kevdel** [comment]
Did you never get this figured out?
I thought the esp-idf framework didn't love doing http things

**Reply by jcam** [reply]
No, I didn't yet. And yes you're right, idf's http_request is still not ready
